src/sdk/pynni/nni/compression/torch/pruning/one_shot.py

```diff
@@ -85,12 +85,32 @@ def calc_mask(self, wrapper, wrapper_idx=None):
             return None
 
 class LevelPruner(OneshotPruner):
-    def __init__(self, model, config_list, optimizer=None):
-        super().__init__(model, config_list, pruning_algorithm='level', optimizer=optimizer)
+    """
+    Parameters
+    ----------
+    model : torch.nn.module
+        Model to be pruned
+    config_list : list
+        Supported keys:
+            - sparsity : This is to specify the sparsity operations to be compressed to.
+            - op_types : Operation types to prune.
+    """
+    def __init__(self, model, config_list):
+        super().__init__(model, config_list, pruning_algorithm='level')
 
 class SlimPruner(OneshotPruner):
-    def __init__(self, model, config_list, optimizer=None):
-        super().__init__(model, config_list, pruning_algorithm='slim', optimizer=optimizer)
+    """
+    Parameters
+    ----------
+    model : torch.nn.module
+        Model to be pruned
+    config_list : list
+        Supported keys:
+            - sparsity : This is to specify the sparsity operations to be compressed to.
+            - op_types : Only BatchNorm2d is supported in Slim Pruner.
+    """
```

Comment:
this docstring is put above `__init__`, @chicm-ms @liuzhe-lz do you think this is a good practice?

Reply:
This is to keep consistent with NAS classes and generate correct docstring in readthedocs.

Reply:
I think this is a good practice since `__init__` is the constructor of the class, I checked pytorch and tensorflow, they put docstring of class before `__init__`,  including the parameters of `__init__`.   I was not aware of this before.  For example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/engine/base_layer.py#L114

Reply:
got it, thanks.